cmd/generate.go

```diff
@@ -84,9 +78,9 @@ func (gc *generateCmd) validate(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "error loading translation files")
 	}
 
-	if gc.apimodelPath == "" {
+	if currentConfig.CLIConfig.Generate.APIModel == "" {
```

Comment:
After playing around with it, it's going to involve a larger refactor of the unit tests backing `cmd` which I don't really have the time to invest in any more. If it's all right, I'd like to punt this until after it's merged and can be refactored down the road since this will still functionally be the same.